src/Simplex/Chat.hs

```diff
@@ -276,9 +276,14 @@ agentSubscriber = do
 subscribeUserConnections :: (MonadUnliftIO m, MonadReader ChatController m) => m ()
 subscribeUserConnections = void . runExceptT $ do
   user <- readTVarIO =<< asks currentUser
+  subscribeNonActivatedConnections user
```

Comment:
Pending?

Reply:
also - let's put it in the end, so it subscribes contacts first (kind of starts faster a bit :)